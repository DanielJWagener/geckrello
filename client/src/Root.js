// Simply creates a Redux store and passes it in to a provider tag.
// This logic was originally in index.js, but we've movied it here to
// make our testing process easier.
import React from "react";
import { Provider } from "react-redux";
import { createStore, applyMiddleware, compose } from "redux";
import reduxThunk from "redux-thunk";
import reducers from "./reducers";

const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

const store = createStore(
  reducers,
  {},
  composeEnhancers(applyMiddleware(reduxThunk))
);

export default ({ children }) => {
  return <Provider store={store}>{children}</Provider>;
};
