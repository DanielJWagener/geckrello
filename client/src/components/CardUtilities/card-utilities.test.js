import { shallow } from "enzyme";
import React from "react";
import CardUtilties from "./card-utilities.component";
import Root from "../../Root";

it("renders CardUtilties component", () => {
  expect(
    shallow(
      <Root>
        <CardUtilties />
      </Root>
    )
  ).toMatchSnapshot();
});
