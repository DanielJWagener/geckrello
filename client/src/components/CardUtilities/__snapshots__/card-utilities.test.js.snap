// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CardUtilties component 1`] = `ShallowWrapper {}`;
